diff -ruN mdns-src/mdns.c mdns-src.patched/mdns.c
--- mdns-src/mdns.c	2026-03-17 21:39:25.273001223 +0100
+++ mdns-src.patched/mdns.c	2026-03-17 21:48:14.689029672 +0100
@@ -669,7 +669,7 @@
 		memset(&sock_addr, 0, sizeof(struct sockaddr_in));
 		sock_addr.sin_family = AF_INET;
 #ifdef _WIN32
-		sock_addr.sin_addr = in4addr_any;
+		// TODO FIXME sock_addr.sin_addr = in4addr_any;
 #else
 		sock_addr.sin_addr.s_addr = INADDR_ANY;
 #endif
diff -ruN mdns-src/mdns.h mdns-src.patched/mdns.h
--- mdns-src/mdns.h	2026-03-17 21:47:14.277529102 +0100
+++ mdns-src.patched/mdns.h	2026-03-17 21:39:26.089006587 +0100
@@ -21,8 +21,8 @@
 
 #include <fcntl.h>
 #ifdef _WIN32
-#include <Winsock2.h>
-#include <Ws2tcpip.h>
+#include <winsock2.h>
+#include <ws2tcpip.h>
 #define strncasecmp _strnicmp
 #else
 #include <unistd.h>
